ddpui/ddpdbt/elementary_service.py

```
@@ -202,6 +203,8 @@ def create_elementary_profile(org: Org):
 
     dbt_project_params = DbtProjectManager.gather_dbt_project_params(org, org.dbt)
 
+    from ddpui.celeryworkers.tasks import run_dbt_commands
```

Ruff (0.8.2)

`ddpui.celeryworkers.tasks.run_dbt_commands` imported but unused

Remove unused import: `ddpui.celeryworkers.tasks.run_dbt_commands`

(F401)